front/src/pods/register-collection/register-collection.component.spec.tsx

```diff
@@ -6,7 +6,8 @@ describe("NameEdit component specs", () => {
   it("should display a label and input elements with empty userName value", () => {
     // Arrange
     const props = {
-      entryCollection: []
+      entryCollection: [],
+      onLoadCollection: () => {}
```

Comment:
missing trainig comma to avoid this line eing marked as changed (as it jsut occurred)